### PR TITLE
App util: stub sceAppUtilDrmOpen.

### PR DESCRIPTION
**Pull request by Zangetsu38**
get access menu for some game.

- For more explain, is just actually it spam sce io open dlc file, because for it dlc is here, so actually it loop(depending game, reallity fighter don't loop) and block boot.
With this so i check if drm content is here, and said file is not here on this case and don't try open it after.

# Result : 
- ONE PIECE Unlimited World Red
image

### CODE REVIEW

File: vita3k/modules/SceAppUtil/SceAppUtil.cpp [hunk 1]
```diff
@@ -21,8 +21,10 @@
 #include <psp2/system_param.h>
 
 #include <host/app_util.h>
+
```
Comment (VelocityRa) [inline vita3k/modules/SceAppUtil/SceAppUtil.cpp]: delete newline

Comment (Zangetsu38) [reply vita3k/modules/SceAppUtil/SceAppUtil.cpp]: Oh, i have just add line or séparateur groupe like other

File: vita3k/modules/SceAppUtil/SceAppUtil.cpp [hunk 2]
```diff
@@ -103,8 +105,16 @@ EXPORT(int, sceAppUtilDrmClose) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppUtilDrmOpen) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceAppUtilDrmOpen, emu::SceAppUtilDrmAddcontId *dirName, SceAppUtilMountPoint *mountPoint) {
+    const auto drm_content_id_path{ fs::path(host.pref_path) / (+VitaIoDevice::ux0)._to_string() / host.io.device_paths.addcont0 / reinterpret_cast<char*>(dirName->data) };;
```
Comment (VelocityRa) [inline vita3k/modules/SceAppUtil/SceAppUtil.cpp]:
```suggestion
    const auto drm_content_id_path{ fs::path(host.pref_path) / (+VitaIoDevice::ux0)._to_string() / host.io.device_paths.addcont0 / reinterpret_cast<char*>(dirName->data) };
```
also, you ran the format script right?

File: vita3k/modules/SceAppUtil/SceAppUtil.cpp (hunk 1 above)
Comment (VelocityRa) [reply vita3k/modules/SceAppUtil/SceAppUtil.cpp]: no. that's not our convention

File: vita3k/modules/SceAppUtil/SceAppUtil.cpp (hunk 2 above)
Comment (Zangetsu38) [reply vita3k/modules/SceAppUtil/SceAppUtil.cpp]: Have est format alone normally is good

**Comment by Zangetsu38** [comment]
done, with also run clang format

File: vita3k/modules/SceCommonDialog/SceCommonDialog.cpp [hunk 3]
```diff
@@ -17,12 +17,15 @@
 
 #include "SceCommonDialog.h"
 
-#include <host/app_util.h>
 #include <dialog/types.h>
+
```
Comment (VelocityRa) [inline vita3k/modules/SceCommonDialog/SceCommonDialog.cpp]: remove all changes in this file.
again, that is not our convention

File: vita3k/modules/SceAppUtil/SceAppUtil.cpp (hunk 1 above)
Comment (VelocityRa) [reply vita3k/modules/SceAppUtil/SceAppUtil.cpp]: ^

File: vita3k/modules/SceCommonDialog/SceCommonDialog.cpp (hunk 3 above)
Comment (Zangetsu38) [reply vita3k/modules/SceCommonDialog/SceCommonDialog.cpp]: eu all change ? this file have my implemntentation drm :/

Comment (Zangetsu38) [reply vita3k/modules/SceCommonDialog/SceCommonDialog.cpp]: i have just run clang format

**Comment by Zangetsu38** [comment]
all pushed like you want

**Pull request merged**